To-do-list/Controller/ViewController.swift

```diff
@@ -9,94 +9,56 @@ import UIKit
 
 class ViewController: UIViewController {
     
-    @IBOutlet weak var tableView: UITableView!
-    
+    private var mainViewModel: MainViewModel?
     private let taskManager = TaskManager()
```

Comment:
remove taskManager from viewController pls

Reply:
Yes, thank You, already done.